amv/README.md

```diff
@@ -0,0 +1,22 @@
+# AMV vocabulary  
+AMV: Algorithm Metadata Vocabulary 
+
+# Algorithm Metadata Vocabulary (AMV), a vocabulary for capturing and storing the metadata about the algorithms (a procedure or a set of rules that is followed step-by-step to solve a problem, especially by a computer) and related objects, such as problem, software programmes. The snag faced by the researchers in the current time is the failure of getting relevant results when searching for algorithms in any search engine. AMV vocabulary is represented as a semantic model and produced as OWL file, which can be directly used by anyone interested to create and publish algorithm metadata as a knowledge graph, or to provide metadata service through SPARQL endpoint.   
+
+
+# Persistent URI 
```

Comment:
Maybe use sub heading for this?
```suggestion
## Persistent URI 
```

Reply:
Yes, agree. Accordingly, updated the file.